provenance.jq

```diff
@@ -0,0 +1,78 @@
+def tags:
+  .source.arches[].tags[],
+  .source.arches[].archTags[],
+  .build.img
+;
```

Comment:
What does this return? Should this function be commented like the others?

Reply:
an array of image tags, updated comment, thanks 👍

Reply:
Does it need square brackets around it?

```console
$ echo '{"a":[0,1,2,3,4],"b":["foo","bar","baz"],"c":"see"}' | jq '.a[], .b[], .c'
0
1
2
3
4
"foo"
"bar"
"baz"
"see"
```

Reply:
Collapsing it back down would be reverting some recent changes, see https://github.com/docker-library/meta-scripts/pull/76#discussion_r1763228481

Reply:
> Does it need square brackets around it?

It doesn't need them but we added them for completeness